static/js/vue-cdr-access/src/translations.js

```diff
@@ -42,12 +42,17 @@ export default {
             available_date: "Available after {available_date}",
             collection_id: "Archival Collection ID",
             contains: "Contains",
+            copied_link: "Download URL, ${text}, copied to clipboard",
+            copied_link_failed: "Unable to copy download URL, ${text}, to clipboard",
```

Comment:
I don't think the `$`'s are necessary, it is displaying a $ in front of the url in the message